### PR TITLE
multiple SMTP on website level

### PR DESCRIPTION
**Pull request by obsergiu**

### CODE REVIEW

File: code/Dotdigitalgroup/Email/Model/Email/Template.php [hunk 1]
```diff
@@ -41,6 +41,20 @@ public function send($email, $name = null, array $variables = array())
...
+            $storeId = null;
+        }
+
+        // Get the current store Id
+        if (isset($variables['store'])) {
+            $storeId = $variables['store']->getStoreId();
+        } else {
+            $storeId = null;
+        }
+
```
Comment (cdiacon) [inline code/Dotdigitalgroup/Email/Model/Email/Template.php]: @obsergiu any reason why there is duplicate code above ? ^^^
thanks.

File: code/Dotdigitalgroup/Email/etc/system.xml [hunk 2]
```diff
@@ -2096,7 +2096,7 @@
                     <frontend_type>text</frontend_type>
                     <sort_order>9000</sort_order>
                     <show_in_default>1</show_in_default>
-                    <show_in_website>0</show_in_website>
+                    <show_in_website>1</show_in_website>
```
Comment (cdiacon) [inline code/Dotdigitalgroup/Email/etc/system.xml]: @obsergiu setting the parent node configuration will not make any effect if the parent section is set to show_in_default only line 2091 !

thanks.

Comment (obsergiu) [reply code/Dotdigitalgroup/Email/etc/system.xml]: @cdiacon sorry about that, was a late push after I migrated the changes from master branch to testing. Thank you for pointing out. Fixed.

**Comment by obsergiu** [comment]
Fixed the typo.

File: code/Dotdigitalgroup/Email/Helper/Transactional.php [hunk 3]
```diff
@@ -14,76 +14,82 @@ class Dotdigitalgroup_Email_Helper_Transactional
 
     /**
      * Transactional Email enabled.
-     * @return bool
+     *
+     * @param null $storeId
+     * @return mixed
```
Comment (cdiacon) [inline code/Dotdigitalgroup/Email/Helper/Transactional.php]: @obsergiu can you change please to return bool type here ?
thanks.

Comment (obsergiu) [reply code/Dotdigitalgroup/Email/Helper/Transactional.php]: @cdiacon changed it.

**Pull request merged**